placementApp/views.py

```diff
@@ -73,6 +82,7 @@ def create(self, request, *args, **kwargs):
             status=status.HTTP_400_BAD_REQUEST,
         )
 
+
 class ApplicationViewSet(
     mixins.CreateModelMixin,
     mixins.RetrieveModelMixin,
```

Comment:
Here Student can create, update, list and delete application
Coordinator can view only
TPO can view and update

So add a permission accordingly

Reply:
TPO and Coordinator